snap/snapcraft.yaml

```
@@ -35,12 +35,12 @@ apps:
     environment: &_environment
       # Prep for Mir
       MIR_SERVER_PLATFORM_PATH: ${SNAP}/usr/lib/${CRAFT_ARCH_TRIPLET_BUILD_FOR}/mir/server-platform
-      __EGL_VENDOR_LIBRARY_DIRS: $SNAP/etc/glvnd/egl_vendor.d:$SNAP/usr/share/glvnd/egl_vendor.d
+      __EGL_VENDOR_LIBRARY_DIRS: /var/lib/snapd/lib/glvnd/egl_vendor.d:$SNAP/etc/glvnd/egl_vendor.d:$SNAP/usr/share/glvnd/egl_vendor.d
```

This will tell glvnd to find `10_nvidia.json`:
```suggestion
      __EGL_VENDOR_LIBRARY_DIRS: /etc/glvnd/egl_vendor.d:/usr/share/glvnd/egl_vendor.d:$SNAP/etc/glvnd/egl_vendor.d:$SNAP/usr/share/glvnd/egl_vendor.d
```